.appveyor.yml

```diff
@@ -0,0 +1,74 @@
+environment:
+#  nodejs_version: "0.12.7"
```

Comment:
Not at all, I guess it was left out when I was playing with different environments trying to avoid virtual environments.